manage-ams-access.md

```diff
@@ -1,103 +1,56 @@
-AMS exposes an HTTP interface. Clients can be anything, whether it's the CLI (amc), the registry (aar) or any client you developed yourself.
-However, talking with AMS **requires** a secure and trusted setup for communications with **TLS** and [certificates](https://en.wikipedia.org/wiki/X.509).
+By default, the Anbox Management Client (AMC) runs on the same machine as the [Anbox Management Service (AMS)](tbd) and connects to it through a UNIX socket.
 
-## Managing certificates
+If you want to control AMS remotely, you can install the AMC on a separate machine and configure it to connect to AMS through a secure HTTP connection.
 
-Through AMS eyes, every client is similar. To establish communication, AMS needs the client certificates, and the client needs AMS certificate.
+## Install the AMC on a separate machine
 
-You can generate certificates using a Certificate Authority or via openssl and self signing it. The former is better suited for multiple clients while the latter is more straight forward.
+To install the AMC on a machine other than the machine running AMS, you must first install it.
 
-### Option 1: Certificate Authority (CA)
+To do so, use the following command:
 
-Certificates authorities are useful for larger teams as you only need to trust a single certificate. 
+    snap install amc
 
-```text
-                         +-----+
-                         | CA  |
-           +-------------+--+--+-------------+
-           |                |                |
-    +------v------+  +------v------+  +------v------+
-    | Certificate |  | Certificate |  | Certificate |
-    +-------------+  +-------------+  +-------------+
+## Install a trusted certificate
 
-    Trusting a CA trusts all its signed certificates as well
-```
+Controlling AMS remotely requires trusted security certificates. You can generate self-signed certificates or use certificates signed by a Certificate Authority. See [Security certificates for remote clients](tbd#security-certificates) for more information.
 
-To setup you own Certificate Authority, you need to generate its own certificate and key
+### Self-signed certificates
 
-```bash
-$ openssl genrsa -out ca.key 2048
-$ openssl req -new -x509 -key ca.key -out ca.crt -days 1024
-```
+To use a self-signed certificate, complete the following steps:
 
-and tell AMS to trust it and verify all subsequents certificates.
+1. Invoke `amc` once on the client machine to generate the certificate.
```

Comment:
Simply invoking `amc` command won't create a key pair on the client machine. People need to explicitly run 
```
amc remote ls
```

Reply:
Thanks, updated!